README.md

```diff
@@ -170,6 +171,7 @@ steps:
 - task: SynopsysSecurityScan@1.0.0
   displayName: 'Black Duck Full Scan'
   condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+  ### Use below configurations to set specific detect environment variables
   env:
     DETECT_PROJECT_NAME: $(Build.Repository.Name)
```

Comment:
detect params for action madhu asked to remove so we can keep that consistent if he suggests for this one too.

Reply:
updated